test/common.go

```diff
@@ -88,3 +94,27 @@ func deleteDefaultVolColl(volcollService *service.VolumeCollectionService) error
 	}
 	return volcollService.DeleteVolumeCollection(*volcoll.ID)
 }
+
+func isFCEnabled(arrayGroupService *service.GroupService) bool {
+	filter := &param.GetParams{}
+	groups, _ := arrayGroupService.GetGroups(filter)
+	group := groups[0]
+	return *group.FcEnabled
+}
+
+func isIscsiEnabled(arrayGroupService *service.GroupService) bool {
+	filter := &param.GetParams{}
```

Comment:
We seem to make a lot of repetitive calls to get the group settings.  If we don't consolidate them, we could at least specify a Fields parameter to limit the response to just the fields we need.  It seems to be just a single field in many cases.

Reply:
Good suggestion. please file an enhancement bug for it. In the interest of time let's keep it .